### PR TITLE
Assert in case node objects are not found and few fixes in drain tests

### PR DESCRIPTION
**Pull request by ebenahar**
Addresses https://github.com/red-hat-storage/ocs-ci/issues/1282

Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/node.py [hunk 1]
```diff
@@ -26,13 +26,17 @@ def get_node_objs(node_names=None):
...
+        return nodes
     else:
-        return [
+        nodes = [
             OCS(**node_obj) for node_obj in node_dicts if (
                 node_obj.get('metadata').get('name') in node_names
             )
         ]
+        assert nodes, "Failed to get the nodes OCS objects"
+        return nodes
```
Comment (sidhant-agrawal) [inline ocs_ci/ocs/node.py]: These lines can be moved outside if-else and then L30 & L31 can be removed.

File: ocs_ci/ocs/node.py [hunk 2]
```diff
@@ -49,7 +53,7 @@ def get_typed_nodes(node_type='worker', num_of_nodes=None):
 
     """
     nodes = get_node_objs()
-
+    assert nodes, "Failed to get the nodes OCS objects"
```
Comment (sidhant-agrawal) [inline ocs_ci/ocs/node.py]: As we are already adding assert in `get_node_objs()` itself, do we still need this one?

File: ocs_ci/ocs/node.py (hunk 1 above)
Comment (ebenahar) [reply ocs_ci/ocs/node.py]: good catch

File: ocs_ci/ocs/node.py (hunk 2 above)
Comment (ebenahar) [reply ocs_ci/ocs/node.py]: no, again, good catch

**Pull request merged**